### PR TITLE
BBrooks/IncreaseASG

### PR DESCRIPTION
**Pull request by cthulhuplus**
3. At the same time, we should bump our ASG’s max size to 24 (in all envs).
4. We may want to tweak our scaling to policy to 2x after 1 minute past 50% CPU usage.

### CODE REVIEW

File: ops/ansible/playbooks-hhsdevcloud/group_vars/all/main.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ data_pipeline_jvm_args: '-Xmx58g'
 data_pipeline_loader_threads: 100  # The service will use x2 this number of DB connections.
 data_server_appserver_jvmargs: '-Xmx58g -XX:MaxMetaspaceSize=4g'
 data_server_db_connections_max: '40'
-data_server_asg_max_size: 7  # Watch out that this (plus extra instances for next deploy) doesn't exceed the max allowed DB connections!
+data_server_asg_max_size: 24  # Watch out that this (plus extra instances for next deploy) doesn't exceed the max allowed DB connections!
```
Comment (njdister) [inline ops/ansible/playbooks-hhsdevcloud/group_vars/all/main.yml]: Deployment in CCS is all done by terraform so I don't think the playbooks in hhsdevcloud are relevant to us.

Comment (RickHawesUSDS) [reply ops/ansible/playbooks-hhsdevcloud/group_vars/all/main.yml]: Right. The line you want to change is the...

module "FHIR_ASG" {
...
asg_config = {
...
    max           = 2*length(local.azs)

Comment (RickHawesUSDS) [reply ops/ansible/playbooks-hhsdevcloud/group_vars/all/main.yml]: In the stateless/main.tf file.

Comment (cthulhuplus) [reply ops/ansible/playbooks-hhsdevcloud/group_vars/all/main.yml]: I reverted the change I made to the ops/ansible/playbooks-hhsdevcloud/group_vars/all/main.yml file and made adjustments to the ops/terraform/modules/stateless/main.tf file.

**Pull request merged**